util/make_new_dif/templates/dif_autogen.c.tpl

```diff
@@ -104,7 +121,7 @@ dif_result_t dif_${ip.name_snake}_init(
 
     static bool ${ip.name_snake}_get_irq_reg_offset(
       dif_${ip.name_snake}_intr_reg_t intr_reg,
-      dif_${ip.name_snake}_irq_t irq,
+      dt_${ip.name_snake}_irq_type_t irq,
```

Comment:
Why the suffix `_type_t`? The `_t` already means `type`, no?

Reply:
done